e2e/virtualmachinevolume.go

```diff
@@ -3,6 +3,7 @@ package e2e
 import (
 	"context"
 	goerrors "errors"
+	"github.com/google/go-cmp/cmp"
```

Comment:
Why this package is needed? looks reflect.deepEqual is better

Reply:
done